walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -30,11 +30,11 @@
     Collection<Event> eventsWithAllAttendees = getEventsWithRequestedAttendees(events, allAttendees);
     Collection<Event> eventsWithRequestedAttendees = getEventsWithRequestedAttendees(events, request.getAttendees());
 
-    Collection<TimeRange> availabeRangesForRequestedAttendees = findAllAvailableTimeRanges(eventsWithRequestedAttendees);
-    Collection<TimeRange> result01 = findViableTimeRangesForRequest(availabeRangesForRequestedAttendees, meetingRequestDuration);
+    Collection<TimeRange> availabeTimeRangesForRequestedAttendees = findAllAvailableTimeRanges(eventsWithRequestedAttendees);
+    Collection<TimeRange> result01 = findViableTimeRangesForRequest(availabeTimeRangesForRequestedAttendees, meetingRequestDuration);
 
-    Collection<TimeRange> availabeRangesForAllAttendees = findAllAvailableTimeRanges(eventsWithAllAttendees);
-    Collection<TimeRange> result02 = findViableTimeRangesForRequest(availabeRangesForAllAttendees, meetingRequestDuration);
+    Collection<TimeRange> availabeRangesForRequestedAndOptionalAttendees = findAllAvailableTimeRanges(eventsWithAllAttendees);
+    Collection<TimeRange> result02 = findViableTimeRangesForRequest(availabeRangesForRequestedAndOptionalAttendees, meetingRequestDuration);
```

Comment:
Still not clear to me what result01 and result02 are supposed to represent.

Reply:
I changed those to be specific  I didnt see anything else that I thought was vague or un-descriptive. Does the new change work ? @rayliu0317